packages/react-dom/src/__tests__/ReactDOMForm-test.js

```diff
@@ -1008,7 +1008,7 @@ describe('ReactDOMForm', () => {
     expect(container.textContent).toBe('0');
 
     await act(() => dispatch('increment'));
-    assertLog(['Async action started [1]']);
+    assertLog(['Async action started [1]', 0]);
```

Comment:
These additional state updates are from the additional `pending` state up date due to re-using the same hook dispatch as `useActionState`.